test/framework/setup.go

```diff
@@ -435,3 +440,8 @@ func CreateCertsForIP(ip string) (cert, key string, err error) {
 
 	return string(keyPair.CertPEM), string(keyPair.KeyPEM), nil
 }
+
+func IsIPv6() bool {
+	_, found := os.LookupEnv(envIPv6)
+	return found
```

Comment:
check the value is `"true"` I'd expect that `IPV6=false` will not trigger V6 test